modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/AbstractKeyValueStorage.java

```diff
@@ -196,6 +223,41 @@ public long getCompactionRevision() {
         }
     }
 
+    @Override
+    public void startCompaction(long revision) {
```

Comment:
can we rename the parameter to `targetRevision` or `compactionRevision`?

Reply:
In general, it is possible.

Reply:
The problem here is that this parameter clashes with the `rev` field, which makes it hard to read the code